lib/net.c

```diff
@@ -54,6 +52,18 @@ uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN] = {
 	0x6a, 0x42, 0xb7, 0x3b, 0xbe, 0xac, 0x01, 0xfa,
 };
 
+/* Function to randomize the default RSS key */
+static void
+randomize_rss_key(void)
+{
+	srand(time(NULL));
+
+	for (int i=0; i<GATEKEEPER_RSS_KEY_LEN; i++) {
+		uint8_t key_randomizer = rand();
+		default_rss_key[i] ^= key_randomizer;
```

Comment:
1. Replace `rand()` with the system call `[getrandom(2)](http://man7.org/linux/man-pages/man2/getrandom.2.html)` to guarantee high entropy of the bits.

2. XORing a constant (i.e. the default RSS key) with a random number is redundant; the result is still a random number. Just copy the random number into the key.